app/js/views/element_call_button_view.js

```diff
@@ -0,0 +1,54 @@
+import { View } from 'components/fxos-mvc/dist/mvc';
+import 'components/gaia-button/gaia-button';
+
+(function(window) {
+'use strict';
+
+var shadowTemplate =
+`<style scoped>
+gaia-button {
+	--button-background: green;
+	--button-background-active: lightgreen;
+	--button-color: white;
+	--button-color-active: white;
+	width: 10rem;
+	min-width: 0 !important;
+	margin: 0.5rem !important;
+}
+</style>
+<gaia-button>Call</gaia-button>`;
+
+var proto;
+
+export default class ElementCallButtonView extends View {
+	constructor(controller, phoneFieldController) {
+		proto = Object.create(HTMLElement.prototype);
+		proto.createdCallback = this._createdCallback;
+		proto._scope = this;
```

Comment:
At first glance it seems strange to have a web-component wrapped in a class. IMO we shouldn't have a `View` class, as this is just mirroring the job of custom-elements (which are effectively 'classes'). Can't 'view' modules just export a custom-element constructor, which you can `new` in JS and inject into DOM?

```js
export default document.registerElement(...)
```